webapp/src/components/ParcelDetailPage/ParcelDetailPage.js

```diff
@@ -54,7 +56,8 @@ export default class ParcelDetailPage extends React.PureComponent {
       districts,
       publications,
       onBuy,
-      onParcelClick
+      onParcelClick,
+      user
```

Comment:
I name it `user` as the current user seeing the Daap because as it is the `parcelDetailPage`, the user is not always the `borrower` (only for mortgages)